integrationTests/testSyncNode.go

```diff
@@ -0,0 +1,171 @@
+package integrationTests
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/consensus/spos/sposFactory"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/mock"
+	"github.com/ElrondNetwork/elrond-go/node"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/block"
+	"github.com/ElrondNetwork/elrond-go/process/smartContract"
+	processSync "github.com/ElrondNetwork/elrond-go/process/sync"
```

Comment:
Is the processSync nickname really needed?

Reply:
no, was copy-pasted. Will remove alias